.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: ShellCheck
+
+on: [push]
+
+jobs:
+  lint:
```

Comment:
@pmarsceill what's a best practice for naming the job?

Reply:
This is just the ID that the system uses. You can optionally include a `name:` definition that is more user-friendly to show up in the UI. Jobs will be shown as `Workflow name / Job name` so the best practice is to make it not look like garbage by using super long strings.

Reply:
This is how it currently shows up:

![image](https://user-images.githubusercontent.com/896475/63278725-d14da680-c275-11e9-806f-48824337b2f4.png)


If you cared, you could do:

```
jobs:
  lint:
    name: Lint
```